<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWindow</class>
 <widget class="QMainWindow" name="UserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1302</width>
    <height>1100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UserWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>1301</width>
      <height>1071</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget{background: url(:/res/user_back.jpg)}</string>
    </property>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>261</x>
       <y>40</y>
       <width>1001</width>
       <height>1001</height>
      </rect>
     </property>
     <widget class="QWidget" name="booksPage">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTableView" name="bookTableView">
         <property name="font">
          <font>
           <family>华文彩云</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="sizeConstraint">
          <enum>QLayout::SetNoConstraint</enum>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="workPage">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QTableView" name="workTableView">
         <property name="font">
          <font>
           <family>华文彩云</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="addWorkButton">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>华文彩云</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>添加工作任务</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeWorkButton">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>华文彩云</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>删除工作任务</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="modifyWorkButton">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>华文彩云</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>修改工作任务</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="recordsPage">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QTableView" name="recordTableView">
         <property name="font">
          <font>
           <family>华文彩云</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="addRecordButton">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>华文彩云</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>添加记录</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeRecordButton">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>华文彩云</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>删除记录</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="modifyRecordButton">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>华文彩云</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>修改记录信息</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="sidebar" native="true">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>221</width>
       <height>985</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#widget{background:url(:/res/user_back.jpg)}</string>
     </property>
     <widget class="QPushButton" name="showBooksButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文彩云</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>显示书籍信息</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>res/tool_button.png</normaloff>
        <normalon>:/res/tool_button.png</normalon>res/tool_button.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="showWorkButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文彩云</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>显示工作任务</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>res/tool_button.png</normaloff>
        <normalon>:/res/tool_button.png</normalon>res/tool_button.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="showRecordsButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>180</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文彩云</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>显示上书记录</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>res/tool_button.png</normaloff>
        <normalon>:/res/tool_button.png</normalon>res/tool_button.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="reloginButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>850</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文彩云</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>重新登陆</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/res/relogin.png</normaloff>:/res/relogin.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="logoutButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>910</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文彩云</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>退出程序</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/res/logout.png</normaloff>:/res/logout.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>26</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="timeLabel">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>1030</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="usernameLabel">
     <property name="geometry">
      <rect>
       <x>711</x>
       <y>1030</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1302</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
